src/Request.js

```diff
@@ -245,17 +246,23 @@ export default class Request {
 
     retry () {
 
-        if (this.retries++ < this._max_retry) {
+        if (this.last_error) {
+            this.errors.push(this.last_error);
+        }
+
+        if (this.retries + 1 < this._max_retry) {
             this.log('warn', 'Retrying request');
```

Comment:
Change this to debug and log more info: ex
1. number of retry
2. error info
3. request info etc.,